docs/en/administration/query_profile_details.md

```diff
@@ -0,0 +1,1366 @@
+
+##### QueryCumulativeScanTime
+
+Description: Cumulative IO time of all Scan nodes. Similar to cumulative operator time, actual execution times of different Scan operations may overlap.
+
+##### QueryPeakScheduleTime
+
+Description: Maximum ScheduleTime metric across all Pipelines.
+
+##### QuerySpillBytes
+
+Description: Number of spilled bytes.
+
+##### ResultDeliverTime
+
+Description: Additional time to transfer results. For query statements, this parameter refers to the time it takes to send data back to the client; for insert statements, it refers to the time it takes to write data to the storage layer.
+
+### Fragment Metrics
+
+##### InstanceNum
+
+Description: Number of all FragmentInstances for this Fragment.
+
+##### InstanceIds
+
+Description: IDs of all FragmentInstances for this Fragment.
+
+##### BackendNum
+
+Description: Number of BEs participating in the execution of this Fragment.
+
+##### BackendAddresses
+
+Description: Addresses of all BEs participating in the execution of this Fragment.
+
+##### FragmentInstancePrepareTime
+
+Description: Time spent in the Fragment Prepare phase.
+
+##### InstanceAllocatedMemoryUsage
+
+Description: Cumulative allocated memory for all FragmentInstances under this Fragment.
+
+##### InstanceDeallocatedMemoryUsage
+
+Description: Cumulative deallocated memory for all FragmentInstances under this Fragment.
+
+### Pipeline Metrics
+
+The relationship between core metrics is illustrated in the following diagram:
+
+- DriverTotalTime = ActiveTime + PendingTime + ScheduleTime
+- ActiveTime = ∑ OperatorTotalTime + OverheadTime
+- PendingTime = InputEmptyTime + OutputFullTime + PreconditionBlockTime + PendingFinishTime
+- InputEmptyTime = FirstInputEmptyTime + FollowupInputEmptyTime
+
+![profile_pipeline_time_relationship](../assets/Profile/profile_pipeline_time_relationship.jpeg)
+
+##### DegreeOfParallelism
+
+Description: Degree of parallelism.
+
+##### TotalDegreeOfParallelism
+
+Description: Sum of degrees of parallelism. Since the same Pipeline may execute on multiple machines, this aggregates all parallelism.
+
+##### DriverPrepareTime
+
+Description: Time to execute Prepare, excluding it from DriverTotalTime.
+
+##### DriverTotalTime
+
+Description: Total execution time of the Pipeline. Excludes time spent in the prepare phase.
+
+##### ActiveTime
+
+Description: Execution time of the Pipeline, including the execution time of each operator and the overall framework overhead, such as time spent on methods like has_output, need_input, etc.
+
+##### PendingTime
+
+Description: Time the Pipeline is blocked from being scheduled for various reasons.
+
+##### InputEmptyTime
+
+Description: Time the Pipeline is blocked due to an empty input queue.
+
+##### FirstInputEmptyTime
+
+Description: Time the Pipeline is first blocked due to an empty input queue. Separating the first time is likely due to dependencies in the Pipeline.
```

Comment:
The first blocking time is separately calculated because the first blocking is mainly caused by Pipeline dependencies.